spec/browser_spec.rb

```diff
@@ -273,4 +273,53 @@
       end
     end
   end
+
+  describe '#response_headers' do
```

Comment:
There are existing specs for `response_headers` in `spec/driver_spec.rb`. I think it would be worth just adding one more example where there's a colon, rather than duplicating the code for booting up a server.